<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherManagerWidget</class>
 <widget class="QWidget" name="TeacherManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>教员管理界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(219, 244, 244);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择您的操作：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>查询学生信息</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>修改学生信息</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>增加学生信息</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>删除学生信息</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>显示当前信息</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>求平均分</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>保存</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>结果展示：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>C语言成绩</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数学成绩</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>英语成绩</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>平均分</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
